components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -313,6 +313,12 @@ export class StatsService {
       return formatted;
     }, []);
 
+    // If last 24 hour interval is selected, exclude start_time and end_time
+    // from requests that return data from 24h search index.
+    if (reportQuery.last24h && requestsLast24h) {
```

Comment:
Looks like `requestsLast24h` is the request here; then what is `reportQuery.last24h` ? (Why does it need both?)